packages/template-retail-react-app/app/layouts/site-layout-home/index.jsx

```diff
@@ -0,0 +1,234 @@
+                ))}
+                {/* A general locale as fallback. For example: "en" if default locale is "en-GB" */}
+                <link
+                    rel="alternate"
+                    hrefLang={site.l10n.defaultLocale.slice(0, 2)}
+                    href={`${appOrigin}${getPathWithLocale(site.l10n.defaultLocale, {
+                        location
+                    })}`}
+                />
+                {/* A wider fallback for user locales that the app does not support */}
+                <link rel="alternate" hrefLang="x-default" href={`${appOrigin}/`} />
+            </Seo>
+
+            <ScrollToTop />
+
+            <Box id="app" display="flex" flexDirection="column" flex={1}>
+                <SkipNavLink zIndex="skipLink">Skip to Content</SkipNavLink>
+
+                <Box {...styles.headerWrapper}>
+                    <Header
+                        onMenuClick={onOpen}
+                        onLogoClick={onLogoClick}
+                        onMyCartClick={onCartClick}
+                        onMyAccountClick={onAccountClick}
+                        onWishlistClick={onWishlistClick}
+                    >
+                        <HideOnDesktop>
+                            <DrawerMenu
+                                isOpen={isOpen}
+                                onClose={onClose}
+                                onLogoClick={onLogoClick}
+                                root={allCategories[DEFAULT_ROOT_CATEGORY]}
+                            />
+                        </HideOnDesktop>
+
+                        <HideOnMobile>
+                            <ListMenu root={allCategories[DEFAULT_ROOT_CATEGORY]} />
+                        </HideOnMobile>
+                    </Header>
+                </Box>
+
+                {!isOnline && <OfflineBanner />}
+                <AddToCartModalProvider>
+                    <SkipNavContent
+                        style={{
+                            display: 'flex',
+                            flexDirection: 'column',
+                            flex: 1,
+                            outline: 0
+                        }}
+                    >
+                        <Box
+                            as="main"
+                            id="app-main"
+                            role="main"
+                            display="flex"
+                            flexDirection="column"
+                            flex="1"
+                        >
+                            <OfflineBoundary isOnline={false}>{children}</OfflineBoundary>
+                        </Box>
+                    </SkipNavContent>
+
+                    <Footer />
+
+                    <AuthModal {...authModal} />
+                </AddToCartModalProvider>
+            </Box>
+        </>
+    )
+}
+
+SiteLayoutHome.propTypes = {
+    children: PropTypes.node
+}
+
+export const getLayout = (page) => {
+    return <SiteLayoutHome>{page}</SiteLayoutHome>
```

Comment:
Similarly, to reduce code duplicates between SiteLayout and SiteLayoutHome, we can rely on props to the layout. Something like this:

```jsx
export const getLayout = (page) => {
    return <SiteLayout title="SiteLayout Home">{page}</SiteLayout>
}

```